wolfssl/wolfcrypt/settings.h

```diff
@@ -1876,11 +1876,13 @@ extern void uITRON4_free(void *p) ;
     #endif
 #endif
 
-#ifdef _MSC_VER
-    #ifndef HAVE_SSIZE_T
-        #include <BaseTsd.h>
-        typedef SSIZE_T ssize_t;
-    #endif
+#if defined(NO_WC_SSIZE_TYPE) || defined(ssize_t)
```

Comment:
The preprocessor doesn't see types when it comes to `defined()`.

Reply:
Actually that's a deliberate test for someone doing `#define ssize_t` elsewhere, in a user or system header.  Just another chance to catch a conflict.

Reply:
Got it.